pkg/services/rpcsrv/server.go

```diff
@@ -2449,6 +2457,10 @@ func getRelayResult(err error, hash util.Uint256) (any, *neorpc.Error) {
 		return result.RelayResult{
 			Hash: hash,
 		}, nil
+	case errors.Is(err, network.ErrSender):
+		return nil, neorpc.WrapErrorWithData(neorpc.ErrNotarySender, err.Error())
+	case errors.Is(err, network.ErrFallbackTransaction):
```

Comment:
Ditto, `network.ErrFallbackValidAfterDepositExpiration` or something like that.

Reply:
Fixed